onebusaway-android/src/main/java/com/joulespersecond/oba/ObaAnalytics.java

```diff
@@ -129,87 +131,88 @@ public static void reportEventWithCategory(String category, String action, Strin
+                    stopDistance = ObaStopDistance.DISTANCE_7;
+                } else {
+                    stopDistance = ObaStopDistance.DISTANCE_8;
+                }
+
+                Tracker tracker = Application.get().getTracker(Application.TrackerName.APP_TRACKER);
+                Tracker tracker2 = Application.get().getTracker(Application.TrackerName.GLOBAL_TRACKER);
+
+                tracker.send(new HitBuilders.EventBuilder()
+                        .setCategory(ObaEventCategory.STOP_ACTION.toString())
+                        .setAction("Stop Id: " + stopId)
+                        .setLabel(stopDistance.toString())
+                        .setValue(1)
+                        .setCustomDimension(1, obaRegionName)
+                        .build());
+                tracker2.send(new HitBuilders.EventBuilder()
+                        .setCategory(ObaEventCategory.STOP_ACTION.toString())
+                        .setAction("Stop Id: " + stopId)
+                        .setLabel(stopDistance.toString())
+                        .setValue(1)
+                        .setCustomDimension(1, obaRegionName)
+                        .build());
             }
-
-            Tracker tracker = Application.get().getTracker(Application.TrackerName.APP_TRACKER);
-            Tracker tracker2 = Application.get().getTracker(Application.TrackerName.GLOBAL_TRACKER);
-
-            tracker.send(new HitBuilders.EventBuilder()
-                    .setCategory(ObaEventCategory.STOP_ACTION.toString())
-                    .setAction("Stop Id: " + stopId)
-                    .setLabel(stopDistance.toString())
-                    .setValue(1)
-                    .setCustomDimension(1, obaRegionName)
-                    .build());
-            tracker2.send(new HitBuilders.EventBuilder()
-                    .setCategory(ObaEventCategory.STOP_ACTION.toString())
-                    .setAction("Stop Id: " + stopId)
-                    .setLabel(stopDistance.toString())
-                    .setValue(1)
-                    .setCustomDimension(1, obaRegionName)
-                    .build());
         }
-
     }
 
     /**
      * For reporting activities on Start
      *
-     * @param activity
+     * @param activity The activity being reported
      */
     public static void reportActivityStart(Activity activity) {
-        Tracker tracker = Application.get().getTracker(Application.TrackerName.APP_TRACKER);
-        tracker.setScreenName(activity.getClass().getSimpleName());
-        tracker.send(new HitBuilders.ScreenViewBuilder().build());
-        Tracker tracker2 = Application.get().getTracker(Application.TrackerName.GLOBAL_TRACKER);
-        tracker2.setScreenName(activity.getClass().getSimpleName());
-        tracker2.send(new HitBuilders.ScreenViewBuilder().build());
+        if (isAnalyticsActive()) {
+            Tracker tracker = Application.get().getTracker(Application.TrackerName.APP_TRACKER);
```

Comment:
@bbodenmiller I don't really understand what you mean by ```have this as a function of ObaAnalytics```. But if you mean that we should implement ```getTracker(Application.TrackerName.APP_TRACKER)``` in the ```ObaAnalytics``` class:

In the Google analytics documentation, they recommend to manage these functions and trackers in the ```Application``` class. 

````
With the new SDK, developers should manage the trackers themselves. To ensure that the
metrics are not over-counted, it is highly recommended that the tracker be created and 
managed in the Application class.
`````